python/tvm/driver/tvmc/frontends.py

```diff
@@ -249,13 +314,12 @@ def suffixes():
         return ["pth", "zip"]
 
     def load(self, path, shape_dict=None, **kwargs):
-        # pylint: disable=C0415
-        import torch
+        tc = import_torch()
```

Comment:
I'd suggest keeping this as `torch` if we can as it's easier to understand as a variable name.